RecoLocalCalo/HGCalRecAlgos/src/RecHitTools.cc

```diff
@@ -104,8 +104,10 @@ std::float_t RecHitTools::getSiThickness(const DetId& id) const {
   auto geom = geom_->getSubdetectorGeometry(id);
   check_geom(geom);
   if( id.det() != DetId::Forward ) {
-    LogDebug("getSiThickness::InvalidSiliconDetid")
-      << "det id: " << id.rawId() << " is not HGCal silicon!";
+    edm::LogWarning("getSiThickness::InvalidSiliconDetid")
```

Comment:
was this change intentional?
there are a lot of messages of the following kind showing up in tests
e.g. https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-21440/25161/runTheMatrix-results/20034.0_TTbar_14TeV+TTbar_14TeV_TuneCUETP8M1_2023D17_GenSimHLBeamSpotFull14+DigiFullTrigger_2023D17+RecoFullGlobal_2023D17+HARVESTFullGlobal_2023D17/step3_TTbar_14TeV+TTbar_14TeV_TuneCUETP8M1_2023D17_GenSimHLBeamSpotFull14+DigiFullTrigger_2023D17+RecoFullGlobal_2023D17+HARVESTFullGlobal_2023D17.log
```
%MSG-w getSiThickness::InvalidSiliconDetid:   HGCalHitCalibration:hgcalHitCalibration 19-Dec-2017 00:14:55 CET  Run: 1 Event: 1
det id: 45288d5b:4 is not HGCal silicon!
```

Reply:
I put it in to indicate the caller to this should be careful about this. Should I make it silent? This should not be called for scintillators which is not the case as you can see this. I am not sure which code needs improvement

Reply:
May I ask to separate this change from this PR which is meant to cover just the CaloCellGeometry signature.

Reply:
@bsunanda there is an open issue about this #20658 - please do not reenable these warning messages in this PR

Reply:
I have removed the warning to LogDebug